pynitrokey/cli/fido2.py

```diff
@@ -405,7 +419,12 @@ def challenge_response(serial, host, user, prompt, credential_id, challenge, udp
 )
 @click.argument("hash-type")
 @click.argument("filename")
-def probe(serial, udp, hash_type, filename):
+def probe(
+    serial: Optional[str],
+    udp: bool,
+    hash_type: Literal["SHA256", "SHA512", "RSA2048"],
```

Comment:
As this is only enforced within the function and not by `click`, shouldn’t this be `str`?  Ideally, we would use an enum and `click.Choice`.

Reply:
going for `str`